ROADMAP.md

```diff
@@ -0,0 +1,123 @@
+
+  *Definition of Done*<br>
+  When most of the API can be used asynchronously.
+
+
+- **Beta availability of the extension in k6 Cloud for all users.**<br>
+
+  *How will we achieve this?*<br>
+  The deployment should be optimized and the extension thoroughly tested before making it available to all users. Frontend changes should be done at this point, and usage costs (CPU, RAM, storage) and pricing details should be determined, followed by public announcements of the availability. Features such as screen capture, video recording, downloading and uploading files should be available.
+
+  *Definition of Done*<br>
+  When all users are eligible to upgrade to a plan that includes browser testing.
+
+
+- **Increase test code coverage; refactor problematic areas of the code base; fix "flaky" tests, linter issues, etc.**<br>
+  This is both a mid and long term goal we'll be constantly focusing on. See issues [#228](https://github.com/grafana/xk6-browser/issues/228), [#241](https://github.com/grafana/xk6-browser/issues/241) and [#58](https://github.com/grafana/xk6-browser/issues/58).
+
+  *How will we achieve this?*<br>
+  By gradually adding more unit and functional tests and fixing issues when the opportunity arises. We won't have long stints where we focus exclusively on this goal, but it's something we'll dedicate as much time as possible to.
+
+  *Definition of Done*<br>
+  The goal is not to reach 100% code coverage, but realistically speaking 80 or 90% should be achievable. Our CI test runs should be stable and flake-free and we should have no linter issues.
+
+
+Long-term goals
+---------------
+
+These are goals achievable after a year, tentatively done in 2024 and beyond.
+
+- **Add support for Firefox and other WebKit-based browsers.**<br>
+  Currently (April 2022) our main focus is supporting Chromium based browsers. This should be expanded to include other browsers as well. The main challenges here will be around CDP and the differences in behavior between browsers.
+
+  *How will we achieve this?*<br>
+  By testing other browsers and fixing issues as they arise.
+
+  *Definition of Done*<br>
+  When the latest Firefox and certain WebKit-based browsers are as well supported as Chromium.
+
+
+- **Feature parity with Playwright and Puppeteer.**<br>
+  Currently our functionality is limited compared to more mature projects like Playwright and Puppeteer. We plan to expand this gradually and reach or exceed the features offered by other browser automation tools.
+
+  *How will we achieve this?*<br>
+  By prioritizing new features to add based on API importance and user feedback. After the short-term stability improvements are done, our main focus will be to add more missing features and close the feature gap we currently have.
+
+  *Definition of Done*<br>
+  When all of the functionality found in other tools is implemented in xk6-browser.
```

Comment:
This might be too broad/vague to be a reachable goal. WDYT?

Reply:
Maybe?
`When we implement all the functions found in other tools, that make sense to include it in the xk6 browser.`

Reply:
Yeah, I changed it to something like that in 89dd838. Thanks.